.scalafmt.conf

```diff
@@ -0,0 +1,16 @@
+maxColumn = 80
+
+style = IntelliJ
```

Comment:
I don't think `defaultWithAlign` is a good idea - my concern here centres on the whitespace alignment that the tool introduces.
Of the remaining options `default` and `intellij`, I'd suggest the latter as it should(?) be more in line with our defacto editor of choice at Cake.